Test/c++/arrays.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %testDafnyForEachCompiler "%s" --refresh-exit-code=0 -- --relax-definite-assignment --spill-translation --unicode-char:false
```

Comment:
Why did this change?

Reply:
Accidentally, but there is no harm in this change either. Reverted it.